activerecord/test/cases/adapters/postgresql/array_test.rb

```diff
@@ -34,14 +34,14 @@ def setup
 
   def test_column
     assert_equal :string, @column.type
-    assert_equal "character varying", @column.sql_type
-    assert @column.array
+    assert_equal "character varying[]", @column.sql_type
```

Comment:
I don't think we should change the result of `Column#sql_type` without a good reason to.